frontend/exporter/src/types/copied.rs

```diff
@@ -2366,6 +2379,15 @@ pub struct AttrItem {
     pub path: String,
     pub args: AttrArgs,
     pub tokens: Option<TokenStream>,
+    /// String representation of the item. This may not exactly match what the user wrote because
```

Comment:
This is redundant with `tokens`, right? I guess it's because it's nicer to process?

Reply:
Sometimes `tokens` is `None` and `as_str` isn't